notebooks/community/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -261,7 +261,7 @@
         "\n",
```

Comment:
<p>GCS -&gt; Cloud Storage (do not use abbreviations for Cloud products)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Fixed</p>